apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue

```diff
@@ -104,14 +106,63 @@
             </SfSelect>
           </label>
         </div>
+
+        <div class="my-2 w-full md:self-end">
+          <button
```

Comment:
Why don't we use this component? https://docs.storefrontui.io/v2/vue/components/accordionitem.html#accordion-group

Reply:
I think that's a question, which needs to be answered by Tim. We just followed the look of the mockups.